src/decomp_2d_constants.f90

```diff
@@ -132,8 +132,8 @@ module decomp_2d_constants
    ! Major and minor version number
    !
    integer, parameter :: D2D_MAJOR = 2
-   integer, parameter :: D2D_MINOR = 2
-   logical, parameter :: D2D_RELEASE = .false.
+   integer, parameter :: D2D_MINOR = 1
+   logical, parameter :: D2D_RELEASE = .true.
```

Comment:
Are these the changes you mentioned @rfj82982? Should I rever to
```
D2D_MINOR = 2
D2D_RELEASE = .false.
```
?

Reply:
Yes we need to revert